### PR TITLE
Issue 3 edit an entry - Kevin Lay

### PR DESCRIPTION
**Pull request by laykevin**
code-journal-issues-32
code-journal-issues-33

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -94,3 +116,28 @@ $newEntry.addEventListener('click', function (event) {
...
+
+$list.addEventListener('click', function (event) {
+  if (event.target.tagName === 'I') {
+    viewSwap('entry-form');
+    for (var o = 0; o < data.entries.length; o++) {
+      if (data.entries[o].entryId.toString() === event.target.closest('[data-entry-id]').getAttribute('data-entry-id')) {
+        data.editing = data.entries[o];
+
+        // console.log(data.entries[o].entryId.toString(), event.target.closest('[data-entry-id]').getAttribute('data-entry-id'));
+        // console.log(data.editing);
```
Comment (shawnkost) [inline js/main.js]: Delete all old code

File: js/main.js [hunk 2]
```diff
@@ -94,3 +116,28 @@ $newEntry.addEventListener('click', function (event) {
...
+    $inputURL.value = data.editing.photoURL;
+    $entryImage.setAttribute('src', $inputURL.value);
+    $inputNotes.value = data.editing.notes;
+    document.querySelector('h2').textContent = 'Edit Entry';
+  }
+  // console.log(data.entries[3].entryId.toString());
+  // // var $liId = event.target.closest('[data-entry-id]');
+  // // console.log($liId);
+  // // console.log($liId.getAttribute('data-entry-id'));
+  // console.log(event.target.closest('[data-entry-id]').getAttribute('data-entry-id'));
```
Comment (shawnkost) [inline js/main.js]: Delete all old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple small changes here!

**Comment by laykevin** [comment]
code-journal-issues-purple

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**